src/fetch/period_slippage.py

```diff
@@ -77,24 +73,26 @@ class SplitSlippages:
     negative: list[SolverSlippage]
     positive: list[SolverSlippage]
 
-    def __init__(self):
+    def __init__(self) -> None:
         self.negative = []
         self.positive = []
 
-    def append(self, slippage: SolverSlippage):
+    def append(self, slippage: SolverSlippage) -> None:
         """Appends the Slippage to the appropriate half based on signature of amount"""
         if slippage.amount_wei < 0:
             self.negative.append(slippage)
         else:
             self.positive.append(slippage)
 
-    def __len__(self):
+    def __len__(self) -> int:
         return len(self.negative) + len(self.positive)
 
     def sum_negative(self) -> int:
+        """Returns total negative slippage"""
```

Comment:
I kinda dislike the enforcement of it. A good funciton name is usually a good enough description, IMHO

Reply:
Your "good function names" are often way to long (entire sentences) and this is what doc strings are for. It would be really nice if you could start using concise function names and put the function description in the functions doc string.